File: core/src/main/java/com/ustadmobile/core/view/DialogResultListener.java
package com.ustadmobile.core.view;

import java.util.Hashtable;

/**
 *
 *
 * Created by mike on 4/16/17.
 */

public interface DialogResultListener {

    /**
     *
     * @param commandId
     * @param args
     */
    void onDialogResult(int commandId, DismissableDialog dialog, Hashtable args);

}

File: core/src/main/java/com/ustadmobile/core/view/DialogResultListener.kt
package com.ustadmobile.core.view

import java.util.Hashtable

/**
 *
 *
 * Created by mike on 4/16/17.
 */

interface DialogResultListener {

    /**
     *
     * @param commandId
     * @param args
     */
    fun onDialogResult(commandId: Int, dialog: DismissableDialog, args: Hashtable<*, *>)

}
